src/ceph-volume/ceph_volume/util/__init__.py

```diff
@@ -30,10 +30,13 @@ def str_to_int(string, round_down=True):
     """
     Parses a string number into an integer, optionally converting to a float
     and rounding down.
+
+    Some LVM values may come with a comma instead of a dot to define decimals.
```

Comment:
does lvm ignore LANG=C and always use the locale or something?

Reply:
That is a really good point @jdurgin . I don't know why this might be, didn't pursue the bug enough to try and find out

Reply:
Did a bit more digging, and it seems like LVM does respect `LANG`. So I am assuming that the machine that was used that triggered this behavior had set it to something else and experienced the bug:

    $ LC_ALL=es_ES lvs -o size
      LSize
       10,00g
       10,00g
       99,00g
      297,51g
        2,00g

    $ LC_ALL=en_US lvs -o size
      LSize
       10.00g
       10.00g
       99.00g
      297.51g
        2.00g

Makes me think that we should try and normalize regardless here

Reply:
this is approach is nice and simple, I'm fine with it here. I've also seen tools force LANG=C when running commands to make output consistent, and wondered if that might avoid future issues here too